openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/DatabaseSchemaRepository.java

```diff
@@ -179,6 +180,18 @@ private void populateDatabase(DatabaseSchema schema) {
         .withServiceType(database.getServiceType());
   }
 
+  @Override
+  public void entityRelationshipReindex(DatabaseSchema original, DatabaseSchema updated) {
+    super.entityRelationshipReindex(original, updated);
+
+    // Update search indexes of assets and entity on databaseSchema displayName change
+    if (!Objects.equals(original.getDisplayName(), updated.getDisplayName())) {
```

Comment:
We are reindexing the whole entity on just displayNameChange, should we just update the disaplayName?

Reply:
this improvement will be handled in a different pr, will link the pr once it is ready